proteus-spring-boot-autoconfigure/src/test/java/io/netifi/proteus/spring/core/ProteusSpringIntegrationTest.java

```diff
@@ -74,6 +66,21 @@
     @Destination(group = "test", destination = "test")
     DefaultExternalIdlClient defaultExternalIdlClient;
 
+    @Autowired
+    @ProteusFactory
```

Comment:
What is the ProteusFactory annotation do?

Reply:
This is one of the things that I'm sure could be better - currently a marker annotation for this check: https://github.com/netifi-proteus/proteus-spring/blob/feature/annotation-tags/proteus-spring-core/src/main/java/io/netifi/proteus/spring/core/annotation/ProteusBeanDefinitionRegistryPostProcessor.java#L86

Reply:
If you want to use one of the tag interfaces do you have to use it?

Reply:
I removed the need for this - I was trying to go with the annotation scan approach we use for generated clients. Now just I check directly if the declared type is a ProteusClientFactory.